internal/nexodus/apiserver.go

```diff
@@ -0,0 +1,134 @@
+package nexodus
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net/url"
+	"syscall"
+
+	"github.com/google/uuid"
+	"github.com/nexodus-io/nexodus/internal/client"
+	"github.com/nexodus-io/nexodus/internal/models"
+	"go.uber.org/zap"
+	"golang.org/x/term"
+)
+
+type ApiServer struct {
+	username      string
+	password      string
+	controllerURL *url.URL
+	ControllerIP  string
+	client        *client.Client
+	logger        *zap.SugaredLogger
+	skipTlsVerify bool
+}
```

Comment:
Adding all of this code, but not removing any?  It would be good to combine that into the same commit to view it as a single change.